libs/ui/src/modals/screens/network-change-confirmation/network-change-confirmation.tsx

```diff
@@ -31,68 +33,76 @@ const changeNetworkRules: AllowsRules[] = [
 ];
 
 export const NetworkChangeConfirmation: FC = () => {
-  const { navigate } = useNavigation();
-  const { name: selectedNetworkName } = useSelectedNetworkSelector();
+  const selectedNetwork = useSelectedNetworkSelector();
+  const showLoader = useShowLoaderSelector();
+  const { name: selectedNetworkName, iconName: selectedNetworkIcon } = useMemo(() => selectedNetwork, []);
```

Comment:
what does this hook do?

Reply:
do not do so much destructurization, it's hard to read.  just use `selectedNetwork.name` and `selectedNetwork.iconName`

Reply:
It remembers the selected network before change

Reply:
<img width="726" alt="image" src="https://user-images.githubusercontent.com/35599200/209947545-46ccc19c-1a1f-4e7f-8e59-bf0e62e09bc2.png">